docs/AWS.md

```diff
@@ -0,0 +1,147 @@
+    listen-peer-urls: http://$private_ipv4:2380,http://$private_ipv4:7001
+  units:
+    - name: etcd2.service
+      command: start
+
+```
+Note: we disable automated reboots for this demo to avoid interrupting the instructions.
+
+Then create the cluster with the following command:
+```
+aws ec2 run-instances \
+  --count 3 \
+  --image-id ami-29734819 \
+  --instance-type t1.micro \
+  --key-name calicokey \
+  --security-groups CalicoSG \
+  --user-data file://cloud-config.yaml
+```
+
+## Installing calicoctl on each node
+Get the public IP addresses of the new instances from the AWS Web Console or by running:
+```
+aws ec2 describe-instances --filter "Name=key-name,Values=calicokey" | grep PublicIpAddress
+```
+
+SSH into each node and run these commands to set up Calico:
+```
+# SSH into a node with the calicokey.pem and username core
+ssh -i calicokey.pem core@<instance IP>
+
+# Download calicoctl and make it executable:
+wget https://github.com/Metaswitch/calico-docker/releases/download/v0.4.5/calicoctl
+chmod +x ./calicoctl
+
+# Grab our private IP from the metadata service:
+export metadata_url="http://169.254.169.254/latest/meta-data"
+export private_ip=$(curl "$metadata_url/local-ipv4")
+
+# Start the calico node service:
+sudo ./calicoctl node --ip=$private_ip
+```
+Then, on any one of the hosts, run this command to create an IP pool with IP-in-IP and NAT enabled:
+```
+./calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
+```
+IP-in-IP alows Calico to route traffic between containers.  NAT allows the containers to make outgoing connections to the internet.
+
+## Create a couple of containers and check connectivity
+On one host, run:
+```
+export DOCKER_HOST=localhost:2377
+docker run -e CALICO_IP=192.168.1.1 -e CALICO_PROFILE=test --name container-1 -tid busybox
+```
+On another host, run:
+```
+export DOCKER_HOST=localhost:2377
+docker run -e CALICO_IP=192.168.1.2 -e CALICO_PROFILE=test --name container-2 -tid busybox
+```
+Then, the two containers should be able to ping each other:
+```
+docker exec container-2 ping -c 4 192.168.1.1
+```
+## Next steps
+Now, you may wish to follow the [getting started instructions for creating workloads](https://github.com/Metaswitch/calico-docker/blob/master/docs/GettingStarted.md#creating-networked-endpoints).
+
+## (Optional) Enabling traffic from the internet to containers
+Services running on containers in AWS can be exposed to the internet using Calico using port mapping iptables NAT rules and an appropriate Calico security profile.  For example, you have a container that you've assigned the CALICO_IP of 192.168.7.4 to, and you have NGINX running on port 80 inside the container. If you want to expose this on port 8000, then you should follow the instructions at https://github.com/Metaswitch/calico-docker/blob/master/docs/AdvancedNetworkPolicy.md to expose port 80 on the container and then run the following commands to add the port mapping:
+
```

Comment:
I think we should take out all references to port mapping here.  There is no need to do port mapping in this example.